### PR TITLE
Make request

### PR DESCRIPTION
**Pull request by dhardiman**
Initial happy case request and parsing for a route. I think this is _slow_ at the moment as bda1a47 was necessary to get it to pass. There's also no unhappy case parsing on this at the moment.

### CODE REVIEW

File: Routing/Geometry.swift [hunk 1]
```diff
@@ -37,3 +37,29 @@ public class BoundingBox: NSObject {
...
+    public override func isEqual(object: AnyObject?) -> Bool {
+        guard let otherObject = object as? BoundingBox else {
+            return false
+        }
+        return self == otherObject
+    }
+}
+
+func ==(lhs: Point, rhs: Point) -> Bool {
+    return lhs.x == rhs.x && lhs.y == rhs.y
```
Comment (drhaynes) [inline Routing/Geometry.swift]: Comparing floating point values with `==` makes me twitchy. Using a delta would give more predictable results.

File: Routing/Geometry.swift [hunk 2]
```diff
@@ -37,3 +37,29 @@ public class BoundingBox: NSObject {
...
+}
+
+func ==(lhs: BoundingBox, rhs: BoundingBox) -> Bool {
+    return lhs.bottomRight == rhs.bottomRight && lhs.topLeft == rhs.topLeft
```
Comment (drhaynes) [inline Routing/Geometry.swift]: Likewise here.

File: Routing/Geometry.swift (hunk 1 above)
Comment (dhardiman) [inline Routing/Geometry.swift]: `Double` in swift conforms to `Equatable`, so I'm assuming it compares it with a delta. Will check the source if I can find it though.

Comment (drhaynes) [inline Routing/Geometry.swift]: For anyone else wondering...
screenshot 2016-03-02 11 15 10

File: Routing/Geometry.swift (hunk 2 above)
Comment (dhardiman) [inline Routing/Geometry.swift]: This one isn't comparing floating point numbers, it's comparing `Point`s, so it's fine. Will update the other one accordingly though.

File: Routing/Geometry.swift (hunk 1 above)
Comment (dhardiman) [inline Routing/Geometry.swift]: Updated

**Comment by dhardiman** [comment]
Updated

**Comment by drhaynes** [comment]
LGTM

**Pull request merged**